Dalamud/Game/Gui/Dtr/DtrBarEntry.cs

```diff
@@ -5,10 +5,79 @@
 
 namespace Dalamud.Game.Gui.Dtr;
 
+/// <summary>
+/// Interface representing a read-only entry in the server info bar.
+/// </summary>
+public interface IReadOnlyDtrBarEntry
+{
+    /// <summary>
+    /// Gets the title of this entry.
+    /// </summary>
+    public string Title { get; }
+    
+    /// <summary>
+    /// Gets a value indicating whether this entry has a click action.
+    /// </summary>
+    public bool HasClickAction { get; }
+    
+    /// <summary>
+    /// Gets the text of this entry.
+    /// </summary>
+    public SeString Text { get; }
+    
+    /// <summary>
+    /// Gets a tooltip to be shown when the user mouses over the dtr entry.
+    /// </summary>
+    public SeString Tooltip { get; }
+    
+    /// <summary>
+    /// Gets a value indicating whether this entry is visible.
+    /// </summary>
+    public bool Shown { get; }
```

Comment:
This one seems to be only managed by the owner of the entry. 

The reason for a separate `IsVisible` flag is because visibility can also be modified via `/xlsettings` by adding some entries to the ignore list.

The `DtrBarService` iterates over the entries, then compares if an entry is in an _ignore-list_ to then just skip over it. 
The `Shown` flag remains `true`, even though the entry is invisible to the user. See https://github.com/goatcorp/Dalamud/blob/bac42ac3f139482fc0062447d364fd06decc583f/Dalamud/Game/Gui/Dtr/DtrBar.cs#L202

Ideally, plugins should have access to a property that exactly mimics the `isHide` variable that is used there.

Reply:
Added a separate prop for that.